src/index.html

```diff
@@ -2,13 +2,37 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav_list">
+        <label>
+          <ul>
+            <li class="links"><a href="#" class="is-active">APPLE</a></li>
```

Comment:
```suggestion
            <li class="links">
              <a href="#" class="is-active">Apple</a>
            </li>
```
one tag per line, make text uppercased with css